From 8ae02d7254e077ecacc23d6fed0b7f844f0aac23 Mon Sep 17 00:00:00 2001
From: Arpit Gupta <arpit@apache.org>
Date: Tue, 11 Jun 2013 19:01:40 +0000
Subject: [PATCH 1396/1475] HADOOP-9625. HADOOP_OPTS not picked up by hadoop command. Contributed by Paul Han

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/trunk@1491907 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit a0aaba387cbe0ca7bd271ce4e6ecc836f4b5dd08)
---
 .../hadoop-common/src/main/conf/hadoop-env.sh      |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/conf/hadoop-env.sh b/hadoop-common-project/hadoop-common/src/main/conf/hadoop-env.sh
index e63c8fb..5836a8a 100644
--- a/hadoop-common-project/hadoop-common/src/main/conf/hadoop-env.sh
+++ b/hadoop-common-project/hadoop-common/src/main/conf/hadoop-env.sh
@@ -45,7 +45,7 @@ done
 #export HADOOP_NAMENODE_INIT_HEAPSIZE=""
 
 # Extra Java runtime options.  Empty by default.
-export HADOOP_OPTS="-Djava.net.preferIPv4Stack=true"
+export HADOOP_OPTS="$HADOOP_OPTS -Djava.net.preferIPv4Stack=true"
 
 # Command specific options appended to HADOOP_OPTS when specified
 export HADOOP_NAMENODE_OPTS="-Dhadoop.security.logger=${HADOOP_SECURITY_LOGGER:-INFO,RFAS} -Dhdfs.audit.logger=${HDFS_AUDIT_LOGGER:-INFO,NullAppender} $HADOOP_NAMENODE_OPTS"
-- 
1.7.0.4

